### ISSUE TITLE
kitty no longer supports Fira Code after updating to 3.1

### ISSUE THREAD

**Issue by cyanic-selkie** [tonsky/FiraCode#1000]
Running `kitty list-fonts` no longer finds Fira Code as a compatible font. This happened after updating from 2.1 to 3.1. Before the update everything worked perfectly.

**Comment by trygveaa** [comment]
This is caused by the spacing property missing:

Version 2
```
$ fc-query /usr/share/fonts/TTF/FiraCode-Regular.ttf | grep spacing
	spacing: 90(i)(s)
```

Version 3
```
$ fc-query /usr/share/fonts/TTF/FiraCode-Regular.ttf | grep spacing
$ 
```

**Comment by idabmat** [comment]
There seems to be a workaround for it : https://github.com/tonsky/FiraCode/issues/840

**Comment by tonsky** [comment]
Thanks, I’ll try to fix this ASAP

**Comment by TheConfuZzledDude** [comment]
The list of characters which don't have a width of 1200 or 0 are:
```
uni1E9E 1199
uni300C 1100
uniFF62 600
uniFF63 600
uni27F2 2400
uni27F3 2400
uni27F4 2400
uni27F5 2400
uni27F6 2400
uni27F7 2400
uni27F8 2400
uni27F9 2400
uni27FA 2400
uni27FB 2400
uni27FC 2400
uni27FD 2400
uni27FE 2400
uni27FF 2400
uni2388 2400
uni2415 1194
```
I believe the only ones which would prevent fontconfig from recognising them as dual spaced are
```
uni300C 1100
uniFF62 600
uniFF63 600
```
Since it doesn't consider them to be "approximately equal" to 1200

**Comment by tonsky** [comment]
Hopefully fixed in 3.1

```
[~/iCloud/ws/FiraCode] fc-query distr/ttf/FiraCode-Regular.ttf | grep spacing
	spacing: 90(i)(s)
```
